recipes/kaleido-core/meta.yaml

```diff
@@ -0,0 +1,71 @@
+{% set name = "kaleido-core" %}
+{% set version = "0.1.0a4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/plotly/Kaleido/releases/download/v{{ version }}/kaleido_minimal_linux_x64.zip  # [linux]
+  sha256: 578bf8c9806553d27a61ddb41d710664bebc099bb77bfb95c76277f7ee093a7b  # [linux]
+
+  url: https://github.com/plotly/Kaleido/releases/download/v{{ version }}/kaleido_mac.zip  # [osx]
+  sha256: 01b2bdb0a343df88857d562139d1c96d72b41ff8fd53b5be62379d9f41e29dea  # [osx]
+
+  url: https://github.com/plotly/Kaleido/releases/download/v{{ version }}/kaleido_win_x64.zip  # [win64]
+  sha256: c8c17c91920833882213dd54ab4a1952966ef8ec0046dfe047671070e35ec2a6  # [win64]
+
+build:
+  number: 0
+  # Remove skip after feedstock creation and sysroot_linux-64 2.17 dependency can be
+  # added
+  skip: True  # [linux]
+  ignore_run_exports:
+    # Chromium loads sqlite dynamically
+    - sqlite  # [linux]
+  missing_dso_whitelist:
+    - /usr/lib/*  # [osx]
+    - /System/Library/Frameworks/*  # [osx]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    # Uncomment after feedstock creation to specify minimum GLIBC version and change
+    # docker image to condaforge/linux-anvil-cos7-x86_64
+    # - sysroot_linux-64 2.17  # [linux]
+  host:
+    - expat   # [linux]
+    - nspr    # [linux]
+    - nss     # [linux]
+    - sqlite  # [linux]
+  run:
+    - expat   # [linux]
+    - nspr    # [linux]
+    - nss     # [linux]
+    - sqlite  # [linux]
+    - fontconfig         # [linux]
+    - fonts-conda-forge  # [linux]
+    - __osx >=10.10      # [osx]
```

Comment:
Are you sure about this? I see that Kaleido is linking to `Metal` which was introduced in 10.11 (at least according to what I see in the SDKs)

Reply:
Here's the info I'm going on:
 - The gni build system in chromium declares `mac_deployment_target = "10.10.0"` (https://github.com/chromium/chromium/blob/15cdafe9a819401befdbf450bc0921fcfd02705b/build/config/mac/mac_sdk.gni#L19)
 - Chrome currently lists 10.10 as the minimum requirement (https://support.google.com/chrome/a/answer/7100626?hl=en)

That said, I don't have a 10.10 system available to test it on so it's fine with me if you'd rather be more conservative to start (i.e. set it to 10.11), and then we lower it back to 10.10 once we've been able to test it on this OS.

Reply:
ok